### ISSUE TITLE
summaryについて

### ISSUE THREAD

**Issue by register1** [hirogami/Tabsets201609#2]
サマリーの結果がそのままで見た目が微妙だと思うので体裁変えたほうがいいと思います。

**Comment by hirogami** [comment]
どうんなふうに？

**Comment by register1** [comment]
試しに書き込んだだけなので今のままでもいいですよ。
あえて言うなら、個人的には1st Qu.が中央にない、r知らない人はぱっと見1st Qu.が四分位と認識しずらいと思うので表記を変える、線で区切って表にする、です。（どうでもいい小さなことですが（笑））
stargazer使えばどうなるでしょうか。

**Comment by register1** [comment]
すいません、stargazerはtexやmarkdownのやつなので無理でした。

**Comment by hirogami** [comment]
そうですね。dplyr::summarizeを使えば好きな要素をタイトル付きで好きな順序に並べられるはずなんですが、エラーがでるんですね。表示部分でデータをユーザーが選択したデータをdata()で渡すんだけど、このdata()がデータフレームじゃないので、dyplyが使えないのだと思います。
これを解決できないので、NROW()をsummaryに足したということです。
まあ、私のスキルが体裁を整えるレベルまでにはまだ遠いです。
